ajax/libs/ng2-bootstrap/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "ng2-bootstrap",
+  "version": "1.0.1-beta.2",
+  "description": "angular2 native bootstrap components",
+  "filename": "ng2-bootstrap.min.js",
+  "keywords": [
+    "angular2",
+    "bootstrap",
+    "angularjs",
+    "twitter-bootstrap"
+  ],
+  "author": {
+    "name": "Dmitriy Shekhovtsov",
+    "email": "valorkin@gmail.com"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/valor-software/ng2-bootstrap.git"
+  },
+  "homepage": "https://github.com/valor-software/ng2-bootstrap#readme",
+  "npmName": "ng2-bootstrap",
+  "npmFileMap": [
+    {
+      "basePath": "/",
+      "files": [
+        "**/*"
```

Comment:
Should not use `"**/*"` here, this is used to include all the files in its npm package.

Looks like we can set basePath to bundles, and change `"**/*"` to `"**/ng2-bootstrap*.+(js|css)"` ?